firecracker/microvm.py

```diff
@@ -400,6 +433,17 @@ async def teardown(self):
         if self.stderr_task:
             self.stderr_task.cancel()
 
+        if self.mounted_fs:
+            logger.debug("Waiting for one second for the VM to shutdown")
+            await asyncio.sleep(1)
+            root_fs = self.mounted_fs.name
+            os.system(f"dmsetup remove {root_fs} {root_fs}_base")
+            if self.use_jailer:
+                os.system(f"umount {self.jailer_path}/dev/mapper")
+                os.system(f"umount {self.jailer_path}/dev/aleph-mapper")
+                os.system(f"rm -rf {self.jailer_path}/dev")
```

Comment:
Use pathlib whenever possible, using `os.system` with string interpolation should be avoided if possible.